### ISSUE TITLE
with rnaseq_fq_ids and busco_lineage, compleasm output in wrong dir

### ISSUE THREAD

**Issue by jbh-cas** [Gaius-Augustus/BRAKER#753]
In trying 3.0.7 with --busco_lineage for the first time I had rnaseq fastq libs so make_bam_file() is called in the braker.pl script.

I found the **mb_download** dir of compleasm in the **rnaseq/hisat2** dir.

Although, since I had 2.4 and not compleasm version 2.5 it did not finish on that run, I suspect the compleasm_genome_out dir would also have been created in hisat2.

Perhaps at make_compleasm_hints() start could do `cd $workDir` or some such. As well might be reasonable to return to $workDir at end of make_bam_file() or whatever the pwd was when the routine began.

Copying the bam output from the failed run and using it in the next attempt with --bam is progressing with the compleasm run.

thanks

**Comment by jbh-cas** [comment]
Additional issue:
compleasm allows lineage without _odb10 extension, but this blows up Augustus/scripts/compleasm_to_hints.py

Following is the code compleasm uses to allow both forms, perhaps it can applied to compleasm_to_hints.py early on.

```
        if not lineage.endswith("_odb10"):
            lineage = lineage + "_odb10"
```

**Comment by KatharinaHoff** [comment]
The _odb10 expansion was fixed in https://github.com/Gaius-Augustus/Augustus/commit/e42b3354b8a7fd02af82364d2ab44dc08bb2a050

**Comment by KatharinaHoff** [comment]
I fixed these issues and pushed an updated docker container.

**Comment by jbh-cas** [comment]
Thank you.  The odb10 thing *is* brittle as I saw you mentioned but both compleasm and BUSCO have the same brittleness. BUSCO even makes sure that if an odb suffix is attached that it is odb10 (and not say odb9) and if it is not there it adds an odb10 suffix (which it calls the datasets_version).

I saw somewhere on the BUSCO github that they expect to transition to odb11 sometime this year. That will require all the programs to handle that roll-over at the same time.
